iceoryx_binding_c/test/moduletests/test_notification_info.cpp

```diff
@@ -91,6 +91,9 @@ class iox_notification_info_test : public Test
         return m_memoryManager.getChunk(chunkSettings);
     }
 
+    static void triggerCallback(iox_sub_t sub IOX_MAYBE_UNUSED)
```

Comment:
You can also get remove the variable name `sub` to indicate to the compiler that you do not use the function argument. This is less code and actually the default way.

Reply:
I think the `unused` attribute is not applicable to types only

Reply:
I think you can remove `IOX_MAYBE_UNUSED` as well and just write `static void triggerCallback(iox_sub_t)`.